examples/icmp_socket.py

```diff
@@ -0,0 +1,57 @@
+import socket
+import sys
+
+from rflx.pyrflx import MessageValue, Package, PyRFLX
+
+
+class ICMPSocket:
+
+    package_icmp: Package
+
+    def __init__(self) -> None:
+        pyrflx = PyRFLX([f"specs/icmp.rflx"])
+        self.package_icmp = pyrflx["ICMP"]
+        self.icmp_data = (
+            b"\x4a\xfc\x0d\x00\x00\x00\x00\x00\x10\x11\x12\x13\x14\x15\x16\x17"
+            b"\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f\x20\x21\x22\x23\x24\x25\x26\x27"
+            b"\x28\x29\x2a\x2b\x2c\x2d\x2e\x2f\x30\x31\x32\x33\x34\x35\x36\x37"
+        )
+
+    def send_icmp_request(self) -> None:
+        icmp_request: MessageValue = self.__create_msg()
+        icmp_socket = None
+        try:
+            icmp_socket = socket.socket(
+                family=socket.AF_INET, type=socket.SOCK_DGRAM, proto=socket.IPPROTO_ICMP
+            )
+        except OSError as e:
+            print("Error while creating socket " + e.strerror)
+            sys.exit()
+        try:
+            icmp_socket.sendto(icmp_request.bytestring, ("localhost", 1))
+        except InterruptedError as e:
+            icmp_socket.close()
+            print("Error while sending icmp request" + e.strerror)
+            sys.exit()
```

Comment:
You could just replace this by `sys.exit(f"Error while sending icmp request {e.strerror}")`. The f-string is better readable, faster and you do not forget to add a space easily. Probably you could also just use `e` instead `e.strerror` in the f-string.

Reply:
changed to f-strings